collector/container/rhel/install.sh

```diff
@@ -3,8 +3,11 @@ set -eo pipefail
 
 # UBI 9 requires confirmation with -y flag.
 microdnf upgrade -y --nobest
-microdnf install -y kmod findutils elfutils-libelf
+microdnf install -y elfutils-libelf
 
 microdnf clean all
-rpm --query --all 'curl' '*rpm*' '*dnf*' '*libsolv*' '*hawkey*' 'yum*' 'findutils' | xargs -t rpm -e --nodeps
+# shellcheck disable=SC2046
+rpm --verbose -e --nodeps $(
```

Comment:
How about removing `--nodeps`

Reply:
I'm not entirely sure why `--nodeps` is used here, it was like this when I found it. Looking into rpms help
```
--nodeps                       don't verify package dependencies
```

So I'm guessing that flag is there to leave dependencies of the affected packages alone. Could you elaborate on why you think it would be worth it to remove it? Is it so dependencies that are no longer used get removed?

Reply:
On second thought, I've managed to break collector by removing a kerberos package that was listed as unused but was still needed, so that might be the entire reason the flag is there.